client/components/data/query-jetpack-credentials-status/README.md

```diff
@@ -0,0 +1,37 @@
+# Query Jetpack Credentials Status
+
+`<QueryJetpackCredentialsStatus />` is a React component which dispatches actions for testing Jetpack Credentials to determinate if they are still valid or not.
```

Comment:
Left a bit of suggestions for improvement here:

```suggestion
`<QueryJetpackCredentialsStatus />` is a React component that dispatches actions for testing Jetpack Credentials to determine whether they are still valid.
```

Reply:
Sounds reasonable, thanks for that one!